_std/macros/mob_properties.dm

```diff
@@ -161,6 +161,14 @@ To remove:
 	SEND_SIGNAL(target, COMSIG_MOB_PROP_INVISIBILITY, old_val); \
 	} while(0)
 
+#define PROP_UPDATE_SIGHT(target, prop, old_val) do {\
+	if(!istype(target, /mob/living)) return; \
+	var/mob/living/_living_mob = target; \
+	var/datum/lifeprocess/sight/_sightprocess = _living_mob.lifeprocesses[/datum/lifeprocess/sight]; \
+	var/datum/lifeprocess/hud/_hudprocess = _living_mob.lifeprocesses[/datum/lifeprocess/hud]; \
+	_sightprocess.process(); \
+	_hudprocess.process(); \
```

Comment:
hud process is for tinting
not sure how expensive this gets for large amounts of players